github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -176,6 +176,91 @@ presubmits:
           path: /var/log/audit
           type: Directory
         name: audit
+  - always_run: false
+    run_before_merge: true
+    annotations:
+      fork-per-release: "true"
+      testgrid-dashboards: kubevirt-presubmits
+    cluster: kubevirt-prow-workloads
+    decorate: true
+    decoration_config:
+      grace_period: 30m0s
+      timeout: 4h0m0s
+    labels:
+      preset-bazel-unnested: "true"
+      preset-dind-enabled: "true"
+      preset-docker-mirror-proxy: "true"
+      preset-shared-images: "true"
+      rehearsal.allowed: "true"
+    max_concurrency: 10
+    name: pull-kubevirt-e2e-kind-ovn-k
+    skip_branches:
+    - release-\d+\.\d+
+    spec:
+      containers:
+      - command:
+        - /usr/local/bin/runner.sh
+        - /bin/bash
+        - -exc
+        -  |
+          mount
+          sysctl fs.inotify.max_user_watches=1048576
+          sysctl fs.inotify.max_user_instances=512
+          dnf install -y git which bison openssl
+          curl -L -O https://go.dev/dl/go1.22.3.linux-amd64.tar.gz 
+          rm -rf /usr/local/go && tar -C /usr/local -xzf go1.22.3.linux-amd64.tar.gz
+          export PATH=$PATH:/usr/local/go/bin
+          git clone https://github.com/qinqon/ovn-kubernetes -b kind-configure-overlay-mtu
+          cd ovn-kubernetes
+          curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.19.0/kind-linux-amd64
+          chmod +x ./kind
+          sudo mv ./kind /usr/local/bin/kind
+          mtu=$(expr $(cat /sys/class/net/eth0/mtu) - 58)
+          export KIND_IMAGE=kindest/node
+          export K8S_VERSION=v1.28.0@sha256:dad5a6238c5e41d7cac405fae3b5eda2ad1de6f1190fa8bfc64ff5bb86173213
+          ./contrib/kind.sh -mtu $mtu --enable-interconnect --num-workers 1 || true
+          export KUBECONFIG=~/ovn.conf
+          echo "==== get pod ==="
+          kubectl get pod -n ovn-kubernetes
+          echo "==== describe pod ==="
+          kubectl describe pod -n ovn-kubernetes
+          echo "==== get event ==="
+          kubectl get event -n ovn-kubernetes
+          echo "==== logs ==="
+          for pod in $(kubectl get pod -n ovn-kubernetes -o name); do
+            echo "POD: $pod"
+            kubectl logs -n ovn-kubernetes $pod --all-containers
+          done
+        image: quay.io/kubevirtci/bootstrap-legacy:v20220810-a8f2e6c
```

Comment:
is this the img we need to pin @qinqon ?

Reply:
This is exactly the same node image as the SRIOV lane, problem is that we will not be able to bump kind and maybe k8s in the furure, but this applys to SRIOV too I think.

Reply:
The underlaying problem is the CRIO version at those images and cgroups.